### PR TITLE
Soarted partography display with descending data

### PR DESCRIPTION
**Pull request by gabriel090**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## 
partograph_desc
Summary

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-clinical-view-app/src/hooks/usePartograph.ts [hunk 1]
```diff
@@ -24,9 +24,19 @@ export function usePartograph(patientUuid: string) {
...
+  });
+  const flattedObs = sortedResults
     .flatMap((encounter) => encounter.obs)
-    .filter((obs) => obs?.concept?.uuid === Progress_UUID);
+    .filter((obs) => obs?.concept?.uuid === Progress_UUID)
+    .sort((a, b) => {
+      const dateA = new Date(a.encounterDatetime).getTime();
+      const dateB = new Date(b.encounterDatetime).getTime();
+      return dateB - dateA;
+    });
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/hooks/usePartograph.ts:39]: Best to do the sorting once.

**Review by donaldkibet — APPROVED** [review]

File: packages/esm-patient-clinical-view-app/src/hooks/usePartograph.ts (hunk 1 above)
Comment (donaldkibet) [reply packages/esm-patient-clinical-view-app/src/hooks/usePartograph.ts:39]: Please address this in a follow up PR merging this in

**Comment by donaldkibet** [comment]
@Murithijoshua do you mind checking the e2e failing test

**Comment by Murithijoshua** [comment]
@donaldkibet okay

**Pull request merged**